pkg/minishift/registration/utils.go

```diff
@@ -26,12 +26,18 @@ import (
 )
 
 type RegistrationParameters struct {
-	Username string
-	Password string
+	Username           string
+	Password           string
+	SkipRegistration   bool
+	SkipUnRegistration bool
 }
 
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipRegistration {
+		log.Debug("User choosed to skip registration")
```

Comment:
I think we need to push this to the console i.e. minishift start should print that `Registration is skipped and user need to manually register the VM`.